src/dotnet/README.md

```diff
@@ -95,13 +90,23 @@ Installing only the .NET Runtime or the ASP.NET Core Runtime. (The SDK includes
 }
 ```
 
+Installing .NET workloads. Multiple workloads can be specified as comma-separated values.
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? You can make these changes in `NOTES.md` instead. Thanks!

Reply:
Good point, forgot about that, I moved the changes to NOTES.